docs/OVERVIEW.md

```diff
@@ -0,0 +1,19 @@
+# Overview
+AmpliPi™ is a multi room/zone home audio controller and amplifier made for whole house audio systems with many zones. It can play up to 4 simultaneous audio streams (Pandora, Spotify, AirPlay, etc) or sources (RCA inputs), each routed to one or many zones, all of which are configurable in real-time using the self-hosted AmpliPi Web App or its underlying REST API. It is expandable up to 36 zones using AmpliPi Expanders.
+
+Your AmpliPi controller includes the following components:
+
+- 1 x AmpliPi Controller
+- 1 x This User Manual
+- 6 x Phoenix connectors for speaker connections
```

Comment:
Look up the proper term to use here (see what other manufactures call them). I have usually seen them called Euro connectors. Also we include a US power cable and Ethernet cable and hopefully soon a manual!

Reply:
We change the lengths and even color based on supplier stock, so probably leave length off description. Great, Euroblock sounds good to me!